pyproject.toml

```diff
@@ -35,17 +34,18 @@ classifiers = [
 poetry_installer = "tox_poetry_installer"
 
 [tool.poetry.extras]
-poetry = ["poetry"]
+poetry = ["poetry", "cleo"]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-poetry = {version = "^1.0.0", optional = true}
-poetry-core = "^1.0.0"
+python = "^3.7"
+cleo = {version = "^1.0.0a5", optional = true, allow-prereleases = true}
+poetry = {version = "1.2.0b3", optional = true, allow-prereleases = true}
+poetry-core = {version = "1.1.0b3", allow-prereleases = true}
 tox = "^3.8.0"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 bandit = "^1.6.2"
-black = {version = "^21.12b0", allow-prereleases = true, python = "^3.7"}
+black = "^22.3.0"
```

Comment:
I'd rather move to the stable black in a separate PR rather than rolling it into this change. The reformatting will make this review more error-prone

Reply:
When I don't update black I get:

```
Traceback (most recent call last):
  File "/home/callek/.cache/pypoetry/virtualenvs/tox-poetry-installer-Fk6da_Gk-py3.7/bin/black", line 8, in <module>
    sys.exit(patched_main())
  File "/home/callek/.cache/pypoetry/virtualenvs/tox-poetry-installer-Fk6da_Gk-py3.7/lib/python3.7/site-packages/black/__init__.py", line 1372, in patched_main
    patch_click()
  File "/home/callek/.cache/pypoetry/virtualenvs/tox-poetry-installer-Fk6da_Gk-py3.7/lib/python3.7/site-packages/black/__init__.py", line 1358, in patch_click
    from click import _unicodefun
ImportError: cannot import name '_unicodefun' from 'click' (/home/callek/.cache/pypoetry/virtualenvs/tox-poetry-installer-Fk6da_Gk-py3.7/lib/python3.7/site-packages/click/__init__.py)
```

But since I could fix it by updating black I didn't delve too deeply into it